t/132-timer-counts.t

```diff
@@ -0,0 +1,73 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use lib 'lib';
+use Test::Nginx::Socket::Lua;
+
+repeat_each(1);
+
+plan tests => blocks() * repeat_each() * 2;
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: running count with no running timers
+--- config
+    location /timers {
+        content_by_lua 'ngx.say(ngx.timer.running_count())';
+    }
+--- request
+GET /timers
+--- response_body
+0
+
+
+
+=== TEST 2: running count with no pending timers
+--- config
+    location /timers {
+        content_by_lua 'ngx.say(ngx.timer.pending_count())';
+    }
+--- request
+GET /timers
+--- response_body
+0
+
+
+
+=== TEST 3: pending count with pending timer
+--- config
+    location /timers {
+        content_by_lua '
+            ngx.timer.at(0.05, function() end)
+            ngx.say(ngx.timer.pending_count())
+        ';
+    }
+--- request
+GET /timers
+--- response_body
+1
+
+
+
+=== TEST 4: max running count
+--- config
+    location /timers {
+        content_by_lua 'ngx.say(ngx.timer.max_running_count())';
+    }
+--- request
+GET /timers
+--- response_body
+256
+
+
+
+=== TEST 4: max pending count
```

Comment:
Please use the [reindex](https://github.com/agentzh/old-openresty/blob/master/bin/reindex) script to automatically fix the test indexes and other style issues in your `.t` files.